qiskit/ignis/verification/topological_codes/fitters.py

```diff
@@ -416,6 +584,200 @@ def get_logical_prob(self, results, algorithm='matching'):
 
         return logical_prob
 
+    def draw_3d_error_graph(self, graph, notebook=False):
+        """Draws a 3d Error Graph.
+
+        Args:
+            graph (retworkx.PyGraph) : Error Graph to be visualised.
+            notebook (bool) : Set True if using Jupyter.
+
+        Returns:
+            list: List of camera position, focal point, and view up.
+            numpy.ndarray: Array containing pixel RGB and optionally
+            alpha values.
+
+        Raises:
+            QiskitError: If pyvista is not installed, or there is
+                invalid input
+        """
+        if not HAS_PYVISTA:
+            raise QiskitError('please install pyvista')
+
+        nodes = np.array(graph.nodes(), dtype='f')
+        edges = []
+        edge_label = []
+        for edge in graph.weighted_edge_list():
+            edges.append(graph[edge[0]])
+            edges.append(graph[edge[1]])
+            edge_label.append(str(abs(edge[2])))
+        edges = np.array(edges, dtype='f')
+        if max(np.array(graph.nodes())[:, 2]) == 0:
+            resize = 1
+        else:
+            resize = float(max(np.array(graph.nodes())[:, 2]))
+        nodes[:, 2] = nodes[:, 2]/resize
+        edges[:, 2] = edges[:, 2]/resize
+        edge_centers = [(edges[i]+edges[i+1])/2
+                        for i in range(0, len(graph.edges())*2, 2)]
+        labels = [str(i)for i in graph.nodes()]
+        # Plotting
+        p = pv.Plotter(notebook=notebook)
+        p.set_background("white")
+        pdata = pv.PolyData(nodes)
+        edata = pv.PolyData(edge_centers)
+        p.add_point_labels(pdata, labels, point_size=10, font_size=10,
+                           text_color='black', point_color='red',
+                           render_points_as_spheres=True,
+                           shape_opacity=0, always_visible=True)
+        p.add_point_labels(edata, edge_label, show_points=False, font_size=10,
+                           text_color='black', shape_opacity=0, always_visible=True)
+        for i in range(0, len(edges), 2):
+            p.add_lines(np.array([edges[i], edges[i+1]]),
+                        width=1, color='black')
+        return p.show()
+
+    def draw_2d_error_graph(self, graph):
+        """Draws a 2d Error Graph.
+
+        Args:
+            graph (retworkx.PyGraph) : Error Graph to be visualised.
+
+        Returns:
+            networkx: A 2-d graph.
+
+        Raises:
+            QiskitError: If matplotlib and networkx is not installed, or there is
+                invalid input
+        """
+        if not HAS_MATPLOTLIB and not HAS_NETWORKX:
+            raise QiskitError('please install pyvista')
+        G = nx.Graph()
```

Comment:
It's not clear to me why you need to use networkx here, especially since the error graph is already a retworkx graph. You can use the retworkx `mpl_drawer` function:  https://qiskit.org/documentation/retworkx/stubs/retworkx.visualization.mpl_draw.html it behaves almost identically to networkx's draw: https://qiskit.org/documentation/retworkx/networkx.html#visualization-functions

Reply:
Thanks. I will change it.

Reply:
Thanks. Changed to retworkx